### PR TITLE
Document a oneliner to only reformat changed files

### PR DESCRIPTION
**Pull request by dopiera**
This change is [Reviewable](https://reviewable.io/reviews/googlecloudplatform/google-cloud-cpp/1324)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -73,6 +73,13 @@ $ find google/cloud -o -name '*.h' -o -name '*.cc' -print0 \
     | xargs -0 clang-format -i
 ```
 
+You might find it convenient to reformat only the files which you actually
+touched:
+```console
+$ git st -s | awk 'NF>1{print $NF}' | grep -E '.*\.(cc|h)$' \
```
Comment (coryan) [inline CONTRIBUTING.md]: `git st` does not work for me, maybe because I do not have an alias?

**Comment by dopiera** [comment]
Crap, I got so used to the alias that I assumed it's a default.

**Comment by coryan** [comment]
BTW, stuff like this you can commit with a [skip ci] in the commit message to save CI bandwidth.

**Comment by dopiera** [comment]
Cool, I didn't know about this.

**Pull request merged**